CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.18.5
+
+## Enhancements
+
+- Enumeration behaviour is now the same as drafter such that the valus that are
```

Comment:
Typo `valus`.

I'd not mention drafter but instead say it was aligned with API Elements 1.0 because a reader of this changelog may not know what drafter is and also the current released versions of drafter do not have fixed typeAttribute.